core-services/prow/02_config/_plugins.yaml

```diff
@@ -1046,6 +1049,21 @@ external_plugins:
     - issue_comment
     - pull_request
     name: needs-rebase
+  openshift-kni/performance-addon-operators:
```

Comment:
This can be the org

Reply:
for now we're looking at enabling per repo in the org. we can reevaluate this later on if we need to. There are repos in the org that aren't using the prow/tide flows and maintainers who may not want them enabled at the moment. We're sharing the org across multiple teams.